src/components/Gradebook/index.jsx

```diff
@@ -74,6 +75,15 @@ export default class Gradebook extends React.Component {
     return `?${queryString.stringify(parsed)}`;
   };
 
+  mapAssignmentTypeEntries = (entries) => {
+    const mapped = entries.map(entry => ({
+      id: entry,
+      label: entry,
+    }));
+    mapped.unshift({ id: 0, label: 'All-Assignments' });
```

Comment:
If you had the label besides the overall dropdown, you can have this entry to be `All`. That should be clear enough